src/order/order.ts

```diff
@@ -60,3 +61,43 @@ export interface GiftCertificateOrderPayment extends OrderPayment {
         remaining: number;
     };
 }
+
+export interface OrderConsignment {
+    shipping: OrderShipping[];
+}
+
+export interface OrderShipping {
+    lineItems: LineItemMap;
+    shippingAddressId: number;
+    firstName: string;
+    lastName: string;
+    company: string;
+    address1: string;
+    address2: string;
+    city: string;
+    stateOrProvince: string;
+    postalCode: number;
```

Comment:
Same question, is this actually a number?

Reply:
Thanks for pointint out.
Just found out a [new doc](https://github.com/bigcommerce/api-specs/blob/master/reference/orders.sf.yml) about the API reponse.
WIll check these types and update this PR soon.

Reply:
@TomA-R I have updated the code. Will add `pickup` consignments once MLI is live.
Would you mind taking another look at this PR?

Reply:
Thanks Peng, lgtm